### PR TITLE
REP-2038 Fixing the NPE caused by no configured media-variants

### PR DESCRIPTION
**Pull request by dmnjohns**

### CODE REVIEW

File: repose-aggregator/components/filters/uri-normalization/src/test/java/org/openrepose/filters/urinormalization/UriNormalizationHandlerFactoryTest.java [hunk 1]
```diff
@@ -53,4 +57,23 @@ public void shouldCreateNewInstanceOfContentNormalizationHandler() throws Except
...
+    public void shouldInitializeMediaTypeListenerWhenNoMediaVariantsProvided() throws Exception {
+        HttpServletRequest mockRequest = mock(HttpServletRequest.class);
+        ReadableHttpServletResponse mockResponse = mock(ReadableHttpServletResponse.class);
+
+        UriNormalizationConfig config = new UriNormalizationConfig();
+        instance.configurationUpdated(config);
+
+        UriNormalizationHandler handler = instance.buildHandler();
+
+        try {
```
Comment (adrianjgeorge) [inline repose-aggregator/components/filters/uri-normalization/src/test/java/org/openrepose/filters/urinormalization/UriNormalizationHandlerFactoryTest.java]: WHy is this in a try?

Comment (adrianjgeorge) [inline repose-aggregator/components/filters/uri-normalization/src/test/java/org/openrepose/filters/urinormalization/UriNormalizationHandlerFactoryTest.java]: Tests automatically fail if they throw an exception that wasn't explicitly mentioned in the test annotation

Comment (dmnjohns) [inline repose-aggregator/components/filters/uri-normalization/src/test/java/org/openrepose/filters/urinormalization/UriNormalizationHandlerFactoryTest.java]: Derp, okay. I fix.

**Comment by adrianjgeorge** [comment]
lets see the tests pass, then i'll merge it

**Pull request merged**